keanacobarde/on-paper
# READ ME AND SMALL STYLING CHANGES

## Description
- Created README and Loom
- Changed the styling of the 'Add Category' button to better balance out the page.

## Related Issue
#16 - Readme is successfully created.

## Motivation and Context
Creating a README is necessary for the documentation of projects. The change in spacing and button size adds to the quality of professionalism of the application.

## How Can This Be Tested?
- Navigate to repo page in GitHub and see whether the README updates.
- Log in and ensure that category button is spaced as needed and is of an appropriate width.

## Screenshots (if appropriate):
N / A 

## Types of changes
